### PR TITLE
fix BlockChaos can't show Chinese name

### PR DESCRIPTION
**Pull request by bn0901**
Signed-off-by: zhengbingxian <zhengbingxian@archeros.com>

### What problem does this PR solve?
https://github.com/chaos-mesh/chaos-mesh/issues/3513

### What's changed and how it works?

Add the Chinese name of blockchaos in ui/app/src/i18n/zh.json

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [x] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.3
  - [ ] release-2.2
  - [ ] release-2.1

### Checklist

CHANGELOG

- [x] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [x] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by bn0901** [comment]
@g1eny0ung , I just updated my PR.

Take a look when you get a chance. Thank you!

File: ui/app/src/i18n/zh.json [hunk 1]
```diff
@@ -84,6 +84,9 @@
       "jvm": {
         "title": "JVM 故障"
       },
+      "block": {
+        "title": "块存储 故障"
```
Comment (g1eny0ung) [inline ui/app/src/i18n/zh.json:88]: There should be no space between the two Chinese phrases.

**Review by g1eny0ung — CHANGES_REQUESTED** [review]

File: ui/app/src/i18n/zh.json (hunk 1 above)
Comment (g1eny0ung) [reply ui/app/src/i18n/zh.json:88]: @YangKeao Do you think this Chinese translation is okay?

Comment (bn0901) [reply ui/app/src/i18n/zh.json]: @g1eny0ung  I've already fixed it, thx !

**Review by STRRL — COMMENTED** [review]
LGTM! 

Thanks for your contribution! :tada:

**Review by STRRL — APPROVED** [review]

**Comment by STRRL** [comment]
/cc @YangKeao 

also ping @YangKeao

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**